packages/design-system/src/components/DateField/SingleInputDateField.tsx

```diff
@@ -40,14 +91,22 @@ const SingleInputDateField = (props: SingleInputDateFieldProps) => {
     toYear,
     ...remainingProps
   } = props;
-  const withPicker = fromDate || fromMonth || fromYear;
+  const withPicker = !!(fromDate || fromMonth || Number.isInteger(fromYear));
   const [pickerVisible, setPickerVisible] = useState(false);
 
+  // Set up change handlers
   function handleInputChange(event) {
     const updatedValue = event.currentTarget.value;
     onChange(updatedValue, DATE_MASK(updatedValue, true));
   }
+  function handlePickerChange(date: Date) {
```

Comment:
I'm not sure, but I _think_ this function is producing a lot of warnings on Storybook. This only happens when the calendar/picker is on the component. I'm guessing there isn't a default date value when this component loads and it's causing this to happen?

![Screen Shot 2022-06-07 at 12 47 08 PM](https://user-images.githubusercontent.com/5159392/172439045-40a5503c-ea79-470b-827c-b775a8477623.png)

Reply:
Oh that makes sense! Then I'm good with this PR - approving!